spec/40-other-protocols.md

```diff
@@ -1,11 +1,9 @@
 # Other protocols
 
-Trace Context propagation is crucial to enable distributed tracing scenarios
-for applications spans multiple components. Http is one of the communication
-protocols used for cross components correlation. Extensions of this
-specification define the format of trace context serialization and
-deserialization for other protocols. Note, that those extensions may be at a
+While trace context is defined for HTTP the authors acknowledge it is also relevant ot other protocols. Extensions of this specification define the format of trace context serialization and
```

Comment:
Spec is called "trace context". I was assuming other protocols may make it into this spec eventually. This change undermines the value of other protocols. Also from editorial perspective, is it typical to refer to "authors" when writing specs?